packages/markdown/remark/src/remark-shiki.ts

```diff
@@ -12,6 +44,13 @@ const remarkShiki = async (theme: shiki.Theme) => {
 			html = html.replace('<pre class="shiki"', '<pre class="astro-code"');
 			// Replace "shiki" css variable naming with "astro".
 			html = html.replace(/style="(background-)?color: var\(--shiki-/g, 'style="$1color: var(--astro-code-');
+			// Handle code wrapping
+			// if wrap=null, do nothing.
+			if (wrap === false) {
```

Comment:
I would really like to not mess with the default style by default, unless we consider them broken. I would expect shiki's default output without changes to be fine / not overflow. Is that not the case / do we consider the default shiki output broken?

Reply:
Honestly, I don't know. I've just mimicked `<Code />` behavior

Reply:
Right, but we don't currently add `overflow-x` in `<Code>` today, right? That's what I'm wondering about, since I hadn't heard this complaint / that a fix for this was needed and I generally like us to match default behaviors when we can.

Reply:
> Right, but we don't currently add overflow-x in <Code> today, right?

Actually, yes. I double checked with a brand new app.

> That's what I'm wondering about, since I hadn't heard this complaint

I'm as confused as you. So, I did a little research (thanks Linus for making git blame) and I found this comment: https://github.com/withastro/astro/pull/1208#issuecomment-911978655

Reply:
lol I did this??? Welp, nevermind then 🤦 I'm suprised that I didn't file this as a bug of Shiki then, maybe its trapped in one of my other issues on that repo.

Either way, thanks for digging! this LGTM then

Reply:
lol hahaha